docs/introduction.md

```diff
@@ -4,145 +4,62 @@
   
-With KubeVela, platform builders now have indefinite flexibilities in designing
-and implementing new capabilities without worrying about what and how to expose
-the new capabilities to the end users. However, the ultimate beneficiaries are
-the application developers:
-- Instead of managing a handful Kubernetes YAML files, only a simple
-  docker-compose style [appfile](./docs/developers/devex/appfile.md) is needed
-  to manage an application in KubeVela. 
-- Out of the box, KubeVela provides a CLI tool to simplify the application
-  management workflow which can be easily integrated with existing CI/CD pipelines.
+With KubeVela, platform builders now finally have the tooling support to design and ship any new capabilities to their end-users with high confidence and turn around time measured in hours. 
 
 ## Comparisons
 
 ### Platform-as-a-Service (PaaS) 
 
-The typical examples are Heroku and Cloud Foundry. They also provide full
-application management capabilities and aim to improve developer experience
-and efficiency. KubeVela shares the same goal but its built-in features are
-much lighter and easier to maintain compared to most of the existing PaaS offerings.
-KubeVela core components are nothing but a set of Kubernetes controllers/plugins.
+The typical examples are Heroku and Cloud Foundry. They provides full application management capabilities and aim to improve developer experience and efficiency. KubeVela shares the same goal but its built-in features are much lighter and easier to maintain compared to most of the existing PaaS offerings. KubeVela core components are nothing but a set of Kubernetes controllers/plugins.
 
-The biggest difference lies in the extensibility. Most PaaS systems enforce
-constraints in the type of supported applications and the supported capabilities.
-They are either inextensible or create their own addon systems maintained by the
-user communities. In contrast, KubeVela is built on top of Kubernetes,
-and all the supported capabilities are implemented by Kubernetes CRD controllers.
-No additional addon system is introduced. A new capability can be installed in
-KubeVela at any time by simply registering the CRD in KubeVela.
+Though the biggest difference lies in the extensibility. 
 
+Most PaaS systems enforce constraints in the type of supported applications and the supported capabilities. They are either inextensible or create their own addon systems maintained by the their own communities. In contrast, KubeVela is designed to fully leverage the Kubernetes ecosystem as its capability pool. Hence, there's no additional addon system is introduced in this project. A new capability can be installed in KubeVela at any time by simply registering the CRD and providing a CUElang template.
+
+This comparison also applies to many other platform agnostic developer tools such as `The Serverless Framework`, etc.
```

Comment:
- Remove `This comparison also applies to many other platform agnostic developer tools such as `The Serverless Framework`, etc. ' ?